src/services/php.js

```diff
@@ -1,74 +1,110 @@
+        loop: false // Disable looping
       }
-    ]);
-    this.selectedVersion = phpVersion;
-    return this;
+    ])
+    this.selectedVersion = phpVersion
+    return this
   },
 
   async handle() {
-    const spinner = new Spinner().start('Installing PHP');
+    const spinner = new Spinner().start('Installing PHP')
     try {
-      await runner.run('sudo add-apt-repository -y ppa:ondrej/php');
-      await runner.run('sudo apt update -y');
-      await runner.run('sudo apt upgrade -y');
-      await runner.run(`sudo apt-get -y install php${this.selectedVersion}`);
-      spinner.succeed(`PHP installed (${spinner.elapsedTime.toFixed(2)}ms)`);
+      await runner.run('sudo add-apt-repository -y ppa:ondrej/php')
+      await runner.run('sudo apt update -y')
+      await runner.run('sudo apt upgrade -y')
+      await runner.run(`sudo apt-get -y install php${this.selectedVersion}`)
+
+      spinner.succeed(
+        chalk.bold.green(`PHP installed ${formatElapsedTime(spinner)}`)
+      )
+
       for (const extension of Object.keys(this.extensions)) {
-        const extensionSpinner = new Spinner().start(`Installing php-${extension}`, { withPrefix: ' - ' });
+        const extensionSpinner = new Spinner().start(
+          `Installing php-${extension}`,
+          { withPrefix: ' - ' }
+        )
         try {
-          await runner.run(`sudo apt-get -y install php${this.selectedVersion}-${extension}`);
-          this.extensions[extension] = true;
-          extensionSpinner.succeed(`php-${extension} installed (${extensionSpinner.elapsedTime.toFixed(2)}ms)`);
+          await runner.run(
+            `sudo apt-get -y install php${this.selectedVersion}-${extension}`
+          )
+          this.extensions[extension] = true
+          extensionSpinner.succeed(
+            chalk.greenBright(
+              `php-${extension} installed ${formatElapsedTime(extensionSpinner)}`
+            )
+          )
         } catch (err) {
-          extensionSpinner.failed(`php-${extension} was not found`);
+          extensionSpinner.failed(
+            chalk.red(`php-${extension} Failed to install!`)
+          )
         }
       }
-      return Promise.resolve();
+      return Promise.resolve()
     } catch (err) {
-      spinner.failed('Failed to install PHP');
-      return Promise.reject(err);
+      spinner.failed('Failed to install PHP')
+      return Promise.reject(err)
     }
   },
 
   async afterInstall() {
-    console.log(chalk.dim('PHP Version: ') + chalk.green(`${this.selectedVersion}`));
-
-    const installedExtensions = Object.entries(this.extensions)
-      .filter(([_, installed]) => installed)
-      .map(([extension]) => extension)
-      .join(', ');
-
-    console.log(chalk.dim('Installed PHP extensions: ') + chalk.green(`[${installedExtensions}]`));
+    console.log(
+      chalk.bold.green('\n PHP Version:') +
+        chalk.green(
+          `${this.selectedVersion} has been installed with all common Extensions.`
```

Comment:
@theizekry I think we need to show installed extensions here and let the user decide weather he needs to install more of them manually or the installed is enough